tools/deployment/package-helm/templates/query-scheduler-deployment.yaml

```
@@ -0,0 +1,96 @@
+          ) | nindent 10 }}
+        - {{- include "clp.waitFor" (dict
+            "root" .
+            "type" "service"
+            "name" "redis"
+          ) | nindent 10 }}
+      containers:
+        - name: "query-scheduler"
+          image: "{{ include "clp.image.ref" . }}"
+          imagePullPolicy: "{{ .Values.image.clpPackage.pullPolicy }}"
+          env:
+            - {{- include "clp.celeryBrokerUrlEnvVar" . | nindent 14 }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)